src/azure-cli-core/azure/cli/core/auth/identity.py

```diff
@@ -94,7 +94,9 @@ def _msal_app_kwargs(self):
         return {
             "authority": self._msal_authority,
             "token_cache": Identity._msal_token_cache,
-            "http_cache": Identity._msal_http_cache
+            "http_cache": Identity._msal_http_cache,
+            # TODO: introduce a config option to allow turning broker off
+            "allow_broker": True
```

Comment:
```suggestion
            "allow_broker": True,
```
:-p

Reply:
Why? Isn't last comma supposed to be omitted?

Reply:
It is less-defined, so there is no official preference that you are "supposed" to follow. However, since Python does allow the extra trailing comma, [I prefer](https://stackoverflow.com/a/13761377) to take advantage of it.